### ISSUE TITLE
[BUG] No crafting recipe for AE2 Press

### ISSUE THREAD

**Issue by BushTrocity** [red2blue/AQM2#187]
Modpack Version - 2.1.2

Minecraft Version - 1.18.1

AE2 Press crafting recipe is missing from the quests. I see where it says to do a quest in "What Sorcery is this" but said quest is not there and REI doesn't list the recipe for the Botania craft.

![Screenshot 2022-03-18 052204](https://user-images.githubusercontent.com/3528732/159070021-a35871fe-52b6-49f0-b57a-f2a55e54432d.png)
![Screenshot 2022-03-18 052216](https://user-images.githubusercontent.com/3528732/159070028-479c34cc-2b5c-41ce-8312-4670c1ae118d.png)
![Screenshot 2022-03-18 052359](https://user-images.githubusercontent.com/3528732/159070031-5fe2ee4d-a423-4976-b637-a3e61ee1c5a7.png)

**Comment by Matiss-Priednieks** [comment]
Doesn't help that meteorites don't spawn.

**Comment by itsdinkd** [comment]
You can ignore the text. You need to do the certus quest to complete the presses. Check the lines and the arrows. 

For crafting, it works. Not sure if you changed any kubejs file but I'm looking at the recipe right now.

**Comment by itsdinkd** [comment]
> Doesn't help that meteorites don't spawn.
Hence the crafting?

**Comment by itsdinkd** [comment]
> Doesn't help that meteorites don't spawn.

Hence the recipe.

**Comment by BushTrocity** [comment]
I did have the thought that maybe i needed to complete up to the press quest to make it show up but i did and still nothing. I gave up and just spawned them in. Where is the recipe file at and i can check to see if it is even in my folders. I have not changed the kubejs files. The only recipes listed under kubejs/data/botania are some brews a blood pendent and laputa shard and i dont see a file/folder for AE2. That just on the client. Im on server and no one has the quest in their book.
